packages/security/lib/module.ts

```diff
@@ -0,0 +1,20 @@
+import { DynamicModule } from '@nestjs/common';
+
+import { SecurityCoreModule } from './security-core.module';
+import { SecurityModuleConfig } from './interfaces';
+import { Consumer } from './consumer';
+
+export class SecurityModule {
+  static withConfig(config: SecurityModuleConfig): DynamicModule {
```

Comment:
@ederfduran please remember me explain it to to you maybe it can help you in the future